student-marks-board/index.js

```diff
@@ -0,0 +1,43 @@
+let userData = [];
+
+function fetchData() {
+	fetch('./data/users.json')
+		.then((response) => response.json())
+		.then((data) => {
+			const users = data.users;
+			userData = users;
+			createUI(users[0]);
+		});
+}
+fetchData();
+
+function createUI(users) {
+	const mainDiv = document.getElementById('tableDiv');
+
+	const ul = document.createElement('ul');
+	mainDiv.appendChild(ul);
+
+    let keys = userData.keys('userData');
+
+    for(let j = 0; j <= keys - 1; j++) {
+        for (let i = 0; i <= users.length - 1; i++) {
+            const li = document.createElement('li');
+            li.setAttribute('class','userLi');
+            li.innerHTML = `<span>${users[i].name}</span><span>${users[i].grade}</span>`;
```

Comment:
Wrong again.. Does your `users[i]` has the `name` property? console log users[i] and debug and check..

Hint: You are not using keys.